src/zeit.zig

```diff
@@ -116,7 +124,11 @@ pub fn loadTimeZone(
     defer dir.close();
     const f = try dir.openFile(loc.asText(), .{});
     defer f.close();
-    return .{ .tzinfo = try timezone.TZInfo.parse(alloc, f.reader()) };
+
+    var buf: [4096]u8 = undefined;
```

Comment:
`find -size 4096c` has several results...meaning this could theoretically be too small. 8192 doesn't return any timezones so that one should be safe.